contracts/mixnet/src/mixnodes/transactions.rs

```diff
@@ -248,7 +277,19 @@ pub(crate) fn try_update_mixnode_config(
             .ok_or(ContractError::NoBondFound)
     })?;
 
-    Ok(Response::new())
+    let mut response = Response::new();
+
+    if let Some(proxy) = proxy {
+        // Returns one_unym proxy had to send in order to execute the contract to contract transaction, this is potentially leaky as anyone can say that they're a proxy,
+        // and they could potentially leak 1 unym per transaction, altough I'm pretty sure transaction fees make that silly.
```

Comment:
Is the 1 unym story documented somewhere (at least as in-line comments)? I only got the brief idea that this is necessary, but I don't currently understand why that is.

Reply:
The only documentation is that comment, would be good to reify is else where

Reply:
Have we actually mentioned this issue to cosmwasm people? Maybe there's a more sane workaround to this?

Reply:
I've not brought it to them, I'll open a thing in their discord